client/src/components/V2/Quiz/Questionaire/Question.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import {
+  Radio,
+  RadioGroup,
+  FormControl,
+  FormControlLabel,
+  FormLabel,
+} from '@mui/material';
+
+function Question({ question, handleChange }) {
+  return (
+    <div>
+      <FormControl>
+        <FormLabel>{question.questionText}</FormLabel>
+        <RadioGroup>
+          <FormControlLabel
+            value={question.answerOptions[0].answerKey}
+            control={<Radio />}
+            label={question.answerOptions[0].answerText}
+            onChange={handleChange}
+          />
+          <FormControlLabel
+            value={question.answerOptions[1].answerKey}
+            control={<Radio />}
+            label={question.answerOptions[1].answerText}
+            onChange={handleChange}
+          />
+          <FormControlLabel
+            value={question.answerOptions[2].answerKey}
+            control={<Radio />}
+            label={question.answerOptions[2].answerText}
+            onChange={handleChange}
+          />
+          <FormControlLabel
+            value={question.answerOptions[3].answerKey}
+            control={<Radio />}
+            label={question.answerOptions[3].answerText}
+            onChange={handleChange}
+          />
```

Comment:
how can you write this to be more DRY (Don't Repeat Yourself)?

Reply:
On it! Will be uploading (hopefully) arid code